x-pack/plugins/cases/public/components/case_view/case_view_page.test.tsx

```diff
@@ -583,5 +566,50 @@ describe('CaseViewPage', () => {
         expect(result.queryByTestId('case-view-alerts-table-experimental-badge')).toBeTruthy();
       });
     });
+
+    describe('description', () => {
+      it('renders the descriptions user correctly', async () => {
+        appMockRenderer.render(<CaseViewPage {...caseProps} />);
+
+        const description = within(screen.getByTestId('description-action'));
+
+        expect(await description.findByText('Leslie Knope')).toBeInTheDocument();
+      });
+
+      it('should display description isLoading', async () => {
+        useUpdateCaseMock.mockImplementation(() => ({
+          ...defaultUpdateCaseState,
+          isLoading: true,
+          updateKey: 'description',
+        }));
+
+        appMockRenderer.render(<CaseViewPage {...caseProps} />);
+
+        await waitFor(() => {
+          expect(screen.getByTestId('description-loading')).toBeInTheDocument();
+          expect(screen.queryByTestId('description-action')).not.toBeInTheDocument();
+        });
+      });
+
+      it.skip('it should persist the draft of new comment while description is updated', async () => {
```

Comment:
Added [github issue ](https://github.com/elastic/kibana/issues/151845)to handle it.